pymeasure/experiment/procedure.py

```diff
@@ -75,6 +78,33 @@ def __init__(self, **kwargs):
                 log.info(f'Setting parameter {key} to {kwargs[key]}')
         self.gen_measurement()
 
+    @staticmethod
+    def parse_columns(columns):
```

Comment:
I don't know whether I wrote tests for that method or not.
Please write tests (or adjust them) for this method.
As a static method, writing tests is straight forward: verify that a certain input produces the expected output. (pytest.mark.parametrize is helpful)

If you have questions, don't hesitate to ask.

Reply:
Yea, I just caught that and pushed a change, I should have run pytest first.

Updated tests based on changes.
Modify numeric check in `Procedure.parsed_columns()` from `isinstance()` to `type()` check, because `isinstance(bool, int)` returns `True`. Did a quick test and getting the same time improvements.